src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+/**
+ * A printable interface for a Person's particulars in the addressbook.
+ * Implementations should guarantee: details are present and not null, field values are validated.
+ */
+public interface Printable {
+
+    String getPrintableString();
```

Comment:
Header comment missing here. Use Returns... as a description and omit @return line.